app/views/course/experience_points_disbursement/new.slim

```diff
@@ -33,12 +33,22 @@
       tr
         th = CourseUser.human_attribute_name(:name)
         th = Course::ExperiencePointsRecord.human_attribute_name(:points_awarded)
+        th
     tbody
+      - display_copy_button = @experience_points_disbursement.experience_points_records.length > 1
       = f.simple_fields_for :experience_points_records do |record_fields|
-        = content_tag_for(:tr, record_fields.object.course_user)
-          td = display_course_user(record_fields.object.course_user)
-          td = record_fields.input :points_awarded, label: false,
-                                   input_html: { class: 'points_awarded' }
+        - course_user = record_fields.object.course_user
+        = content_tag_for(:tr, course_user)
+          td
+            = link_to display_course_user(course_user),
```

Comment:
Random qn - when I display a link with a student's name, I'd always redirect that to the student's home page. Not sure if it is intuitive if we click on the `course_user` to go to their `exp_records`? 

If there is a use case to check the `exp_records` of a `course_user`, maybe consider having `Student Name (Experience Points Records)` then link that to the `(Experience Points Records)` text?